frontend/src/features/collections/collections-add.ts

```diff
@@ -209,13 +176,15 @@ export class CollectionsAdd extends BtrixElement {
               this.combobox.show();
             }
           }}
-          @sl-input=${this.onSearchInput as UnderlyingFunction<
-            typeof this.onSearchInput
-          >}
+          @sl-input=${(e: SlInputEvent) => {
+            const input = e.target as SlInput;
+
+            this.searchByValue = input.value.trim();
+          }}
```

Comment:
Chances are it's not a huge issue right now, but flagging that perf issues might come up without debouncing this, especially for orgs with a large number of collections, because searching fuse blocks the main thread, so this might result in input latency as you're typing.

Reply:
Good call out, since fuse search happens in the render block there's probably something clever we could do with `scheduleUpdate`. Moving the search results to a task should achieve something similar (which wasn't clear/fully implemented in my previous changes, per your feedback) as well. I'll add a TODO comment.